lib/rubocop/cop/rspec/yield.rb

```diff
@@ -26,7 +26,7 @@ class Yield < Base
         # @!method block_call?(node)
         def_node_matcher :block_call?, '(send (lvar %) :call ...)'
 
-        def on_block(node)
+        def on_block(node) # rubocop:disable InternalAffairs/NumblockHandler
```

Comment:
this also needs support for numblocks

Reply:
Perhaps, but I think that this cop considers an offense when passing a block argument, so it is always a not offense with numbered parameters.